src/cases/case-feature.routes.ts

```diff
@@ -133,6 +134,10 @@ export const ROUTES: Routes = [
           data: {
             title: 'Case Details'
           }
+        },
+        {
+          path: 'role-access',
```

Comment:
I have put this /role-access as a base path not under /case in the base pr, could you remove this under /case?